packages/framework/build.xml

```
@@ -1160,4 +1161,18 @@
         <property name="yaml-standards.args" value="${yaml-standards.args} --fix" override="true"/>
         <phingcall target="yaml-standards"/>
     </target>
+
+    <target name="stylelint" description="Run style linter.">
+        <exec executable="${path.root}/${path.stylelint.executable}" logoutput="true" passthru="true" checkreturn="true" dir="${path.root}">
```

```suggestion
        <exec executable="${path.stylelint.executable}" logoutput="true" passthru="true" checkreturn="true" dir="${path.root}">
```

see comment near the defined property

---

Fixed